### DISCUSSION TITLE
单独创建的服务（通过项目自动生成工程生成）需要使用redis怎么用？

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#256] [02.提问]
<img width="508" alt="image" src="https://user-images.githubusercontent.com/6092525/227869266-267b8e63-5b09-42d7-9ee3-8685e7f7be25.png">
controller引入这个 可以直接用吗 ？还是需要额外的配置

**Comment by zuihou** [comment]
依赖了 lamp-cache-stater 就可以。

自己试试不就知道了吗。
